addons/mass_mailing_sale/views/mass_mailing_views.xml

```diff
@@ -10,14 +10,16 @@
                     type="object"
                     icon="fa-usd"
                     class="oe_stat_button"
-                    attrs="{'invisible': ['|', ('state', 'in', ('draft','test')), ('mailing_model_name', '!=', 'sale.order')]}" >
+                    groups="sales_team.group_sale_salesman"
```

Comment:
Shouldn't we put groups on the fields directly ?

Shouldn't we add invoicing groups ?

Reply:
As discussed, invoicing groups don't make a lot of sense here.
But I have added the groups on the fields directly as requested.